chord_metadata_service/patients/api_views.py

```diff
@@ -92,9 +95,20 @@ def list(self, request, *args, **kwargs):
                     []
                 )
             )
-            return Response(build_search_response(list(qs), start))
-
-        return super(IndividualViewSet, self).list(request, *args, **kwargs)
+            experiments_with_biosamples = build_experiments_by_subject(biosamples_experiments_details)
+            results = [
+                {
+                    "subject_id": data["subject_id"],
+                    "alternate_ids": data["alternate_ids"],
+                    "num_experiments": data["num_experiments"],
+                    "biosamples": data["biosamples"],
```

Comment:
instead of hard-coding each field to copy them over, maybe use 
```python
{
    **data,
    "experiments_with_biosamples": experiments_with_biosamples.get(data["subject_id"], []),
}
```
to copy the existing contents + add the new field